pointillism-port.patch

```diff
@@ -0,0 +1,116 @@
+diff --git a/activity.py b/activity.py
+index d91ea44..a4a499d 100755
+--- a/activity.py
++++ b/activity.py
+@@ -1,4 +1,4 @@
+-#! /usr/bin/env python
++#! /usr/bin/env python3
+ # -*- coding: utf-8 -*-
+ #
+ # Pointillism
+@@ -24,13 +24,19 @@
+ 
+ import puntillism
+ from gettext import gettext as _
+-from sugar3.datastore import datastore
++# from sugar3.datastore import datastore
+ from sugar3.graphics.objectchooser import ObjectChooser
+ from sugar3.graphics.toolbutton import ToolButton
+ from sugar3.activity.widgets import StopButton
+-from sugar3.activity.widgets import ActivityToolbarButton
++# from sugar3.activity.widgets import ActivityToolbarButton
+ from sugar3.graphics.toolbarbox import ToolbarBox
+-from sugar3.activity import activity
++# from sugar3.activity import activity
++import shutil
++import sys
++from sugarapp.widgets import SugarCompatibleActivity
++from sugarapp.widgets import ExtendedActivityToolbarButton
```

Comment:
You don't need a ExtendedActivityToolbarButton since the app does not save/restore states.